Sources/CallMacroNodes.swift

```diff
@@ -6,10 +6,15 @@
 
 import Stencil
 
-struct CallableBlock: NodeType {
+struct CallableBlock {
   let parameters: [String]
   let nodes: [NodeType]
 
+  init(parameters: [String], nodes: [NodeType], token: Token? = nil) {
+    self.parameters = parameters
+    self.nodes = nodes
```

Comment:
Not sure I get the use of the `token` parameter here given it's unused?

Reply:
It's mandatory for the new error functionality (https://github.com/stencilproject/Stencil/pull/167).